examples/metrics-ingestion-otel/docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: '3'
+services:
+  data-prepper:
+    image: opensearchproject/data-prepper
+    container_name: data-prepper
+    volumes:
+      - ./metric_pipeline.yaml:/usr/share/data-prepper/pipelines/metric_pipeline.yaml
+      - ../data-prepper-config.yaml:/usr/share/data-prepper/config/data-prepper-config.yaml
+    ports:
+      - 2021:2021
+      - 21891:21891
+      - 4900:4900
+    expose:
+      - "2021"
+      - "4900"
+      - "21891"
+    networks:
+      - opensearch-net
+    depends_on:
+      - opensearch
+  opensearch:
+    container_name: opensearch
+    image: docker.io/opensearchproject/opensearch:latest
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true # along with the memlock settings below, disables swapping
+      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # minimum and maximum Java heap size, recommend setting both to 50% of system RAM
+      - "OPENSEARCH_INITIAL_ADMIN_PASSWORD=Developer@123"
```

Comment:
We should use the same password in all of our examples. Originally I was suggesting `yourStrongPassword123!` as the [initial blog post](https://opensearch.org/blog/replacing-default-admin-credentials/) used that.

But, I did just try typing it and it is a bit of a pain. :)

Should we update all usages to `Developer@123` instead?

Reply:
I would vote for using `Developer@123` in all examples. Some (trace-analytics) are still trying to configure `admin`, which is rejected by OpenSearch as too weak.

Reply:
Sounds good!